the constant 69420 does not fit into Type (which is 1 signed byte)
so it is not a constant expression

--- a/components/sceneutil/texturetype.hpp
+++ b/components/sceneutil/texturetype.hpp
@@ -18,7 +18,7 @@ namespace SceneUtil
         {
         }
 
-        static const osg::StateAttribute::Type AttributeType = static_cast<osg::StateAttribute::Type>(69420);
+        static const osg::StateAttribute::Type AttributeType = static_cast<osg::StateAttribute::Type>(112);
         META_StateAttribute(SceneUtil, TextureType, AttributeType)
 
         bool isTextureAttribute() const override { return true; }
